### PR TITLE
Removes promise

### PR DESCRIPTION
**Pull request by AdamBrodzinski**
Removes the unneeded promises. I ran though the app in GraphiQL to make sure all the queries and mutations still work as expected.

Thanks for creating this example app btw!

### CODE REVIEW

File: server/schema.js [hunk 1]
```diff
@@ -16,19 +13,14 @@ const query = new GraphQLObjectType({
     recentPost: {
       type: BlogPost,
       resolve() {
-        const coll = Collections.posts.rawCollection();
-        return sleep(200)
```
Comment (arunoda) [inline server/schema.js:20]: Shall we keep these `sleep()` calls. They help us to show the latency compensation part. 

Use `Meteor._sleepFor(200)`

**Comment by AdamBrodzinski** [comment]
Sure i'll put them back in tomorrow

File: server/schema.js (hunk 1 above)
Comment (lieuwex) [inline server/schema.js:20]: @AdamBrodzinski could you also give it a comment? :) I didn't get the delay at first.

**Comment by AdamBrodzinski** [comment]
@lieuwex  yea I agree, i'll add comments next to the call.
@arunoda  i'll just use `Meteor._sleepForMs(200)` since we have fibers now, if that's ok.
PR coming tonight

**Comment by arunoda** [comment]
Yeah! That's great.

**Comment by AdamBrodzinski** [comment]
commit 3fac6c4 adds sleep back in.

**Comment by arunoda** [comment]
Awesome.

**Pull request merged**